src/modules/overworld_locations/clayport/clayport.tscn

```diff
@@ -1,17 +1,31 @@
-[gd_scene load_steps=17 format=3 uid="uid://b1ucpnb2u1isc"]
+[gd_scene load_steps=22 format=3 uid="uid://b1ucpnb2u1isc"]
 
 [ext_resource type="Script" uid="uid://eiutp4wmfx4t" path="res://modules/overworld_locations/clayport/clayport.gd" id="1_q21dp"]
 [ext_resource type="Resource" uid="uid://cbu6b1qnaw05f" path="res://modules/minigame/earth_potato_herding/data/earth_potato_herding_data.tres" id="2_2ieeh"]
 [ext_resource type="PackedScene" uid="uid://oatd3ruxn0qu" path="res://modules/debug/debug_popup.tscn" id="3_1ftrw"]
 [ext_resource type="Resource" uid="uid://s7k08dgu5hxn" path="res://modules/minigame/earth_tower_tumble/data/earth_towers_data.tres" id="3_wnqv0"]
 [ext_resource type="Script" uid="uid://bniv0kd24eb7w" path="res://modules/debug/debug_button.gd" id="4_i3tw0"]
 [ext_resource type="Resource" uid="uid://ciyrdxjs03cvb" path="res://modules/minigame/water_diving/data/water_diving_data.tres" id="4_ue7jc"]
-[ext_resource type="Resource" path="res://modules/minigame/water_target_fishing/data/water_fishing_data.tres" id="5_0icjx"]
-[ext_resource type="Resource" path="res://modules/minigame/common/debug/overworld.tres" id="7_2h4su"]
+[ext_resource type="Resource" uid="uid://c7vragnlq8udw" path="res://modules/minigame/water_target_fishing/data/water_fishing_data.tres" id="5_0icjx"]
+[ext_resource type="PackedScene" uid="uid://cccmrn0cfcn0s" path="res://modules/minigame/water_rowing_rapids/water_rowing_rapids.tscn" id="6_jn4u6"]
+[ext_resource type="Resource" uid="uid://b50tc478quiok" path="res://modules/minigame/common/debug/overworld.tres" id="7_2h4su"]
+[ext_resource type="Resource" uid="uid://td664oe8wk42" path="res://modules/essence/resources/water.tres" id="7_squwu"]
+[ext_resource type="Script" uid="uid://mwcjys5dkgf4" path="res://modules/minigame/common/minigame_data.gd" id="7_ybwht"]
```

Comment:
I'm not sure if clayport needs this but we can dig into it on a future PR.

Reply:
I think... there is something weird happening here for sure. the minigame data is defined in clayport's scene and not using the one in the filesyste

but yeah, I'll fix this since I'm already looking in this area to help nox out